docs/1 Introduction/1 Introduction.md

```diff
@@ -31,19 +31,15 @@ The technology stack is being used by projects such as:
 
 [See the full list here](https://cosmonauts.world/).
 
-The basis of the consensus protocol, network management and blockchain initialisation has been proven to be reliable and competitive. From there, only your imagination can stop you from what can be built with the technology stack.
-
 ## Modules
 
-What empowers building blockchains and your own application on top of it are the Cosmos modules. Modules are developed by companies and communities around the Cosmos SDK. Some of the modules are crucial to build a basic blockchain while others enable new features for a blockchain.
-
-Each of the running blockchains most of the time have at least one module developed on their own to enable the features it specialises on. These standards solve the clearity of a project and the understanding of all the different parts that create a specific blockchain application.
+Cosmos modules are the foundational building blocks for building a blockchain. Modules can be created and shared with anyone. Each module plugs into the Cosmos SDK. Some of the modules are core building blocks for creating blockchains, while other modules enable new features.
 
-The most basic set of modules that are created with starport are `auth`, `bank`, `staking`, `params` and `supply`. If you would also add the `wasm` module, this would allow you to upload Webassembly smart contracts on your blockchain application. The `evm` module enables the Ethereum EVM to be used on your blockchain. Each module comes with a clear documentation and codebase. If you wanted changes to a specific module, you can fork the module and change what suits your blockchain application better.
+Many of the live blockchains use multiple Cosmos modules. The foundational modules for starport are: `auth`, `bank`, `staking`, `params` and `supply`. We also recommend adding the `wasm` or the `evm` module, this allows you to deploy Web Assembly smart contracts to your blockchain. The `evm` module enables the Ethereum EVM to be used in your blockchain. Each module comes with clear documentation and codebase. If you wanted to make changes to a specific module, you can fork the module and change what suits your use case better.
 
 ## Summary
 
-- Starport lets you create, develop, host in the cloud and manage the initiation of the blockchain.
-- Starport and Cosmos are written in Go
-- Today, Cosmos SDK has a unique position worldwide of the most successful blockchains being built with it.
-- A combination of different modules with the Cosmos SDK will create different technology opportunities for a blockchain.
+- Starport lets you create, develop, and build a blockchain.
+- Starport and Cosmos are written in Go.
+- Today, Cosmos SDK has a unique position worldwide as one of the most successful blockchains.
```

Comment:
Missing word?

I would say
"Today, Cosmos SDK has a unique position worldwide as one of the most successful blockchain framework."
or 
"Today, Cosmos SDK has a unique position worldwide, some of the most successful blockchains are built with it."